server/sonar-db-dao/src/main/java/org/sonar/db/rule/RuleDao.java

```diff
@@ -47,11 +47,11 @@
   public Optional<RuleDto> selectByKey(DbSession session, String organizationUuid, RuleKey key) {
     RuleDto res = mapper(session).selectByKey(organizationUuid, key);
     ensureOrganizationIsSet(organizationUuid, res);
-    return Optional.fromNullable(res);
+    return Optional.ofNullable(res);
   }
 
   public Optional<RuleDefinitionDto> selectDefinitionByKey(DbSession session, RuleKey key) {
-    return Optional.fromNullable(mapper(session).selectDefinitionByKey(key));
+    return Optional.ofNullable(mapper(session).selectDefinitionByKey(key));
   }
 
   public java.util.Optional<RuleMetadataDto> selectMetadataByKey(DbSession session, RuleKey key, OrganizationDto organization) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this fully qualified name with "Optional" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1942)